packages/safety-suite-api/src/api/models/queries.ts

```diff
@@ -368,6 +368,21 @@ export function runEnforcementsReportExport(
   });
 }
 
+export interface TurnoverReportFilters {
+  /**
+   * Report start timestamp.
+   */
+  dateFrom: number;
+  /**
+   * Report till timestamp.
+   */
+  dateTo: number;
+  /**
+   * `terminal` group id.
+   */
+  terminalId: number;
```

Comment:
Add `timezone?: string` to more accurately reflect the backend [`TurnoverInput`](https://github.com/idelic-inc/safety-suite/blob/master/backend/app/com/idelic/safetysuite/models/exports/turnover/TurnoverInput.scala#L5) model.